### PR TITLE
Feature/skeleton loader

### PR DESCRIPTION
**Pull request by nikita1tup**

### CODE REVIEW

File: src/views/AccountsItem.vue [hunk 1]
```diff
@@ -69,9 +69,13 @@
           />
         </template>
         <template v-else>
-          <div>
-            <p v-if="isLoading" class="empty mg-t32">Loading…</p>
-            <p v-else class="empty mg-t32">No items yet</p>
+          <SkeletonTable
+            v-if="isLoading"
+            :header-titles="headerTitles"
+            :tableSize="10"
```
Comment (ArtSuslov) [inline src/views/AccountsItem.vue:75]: use kebab-case and no v-bind with primitives

File: src/views/ValidatorsView.vue [hunk 2]
```diff
@@ -267,8 +325,18 @@ export default defineComponent({
...
+      isActive: boolean
+    }) => {
+      if (validator.status === 3) {
+        return validator.isActive ? 'success' : 'error'
+      } else {
+        return 'inactive'
+      }
+    }
     onMounted(async () => {
+      window.addEventListener('resize', updateWidth)
```
Comment (ArtSuslov) [inline src/views/ValidatorsView.vue:339]: removing event listeners is a [good practice](https://stackoverflow.com/questions/6033821/do-i-need-to-remove-event-listeners-before-removing-elements)

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

File: src/helpers/helpers.ts [hunk 3]
```diff
@@ -248,3 +247,14 @@ export enum VALIDATOR_STATUS_TYPE {
   success = 'success',
   error = 'error',
 }
+export const duplicateArrayNCount = (
```
[the remark this answers is not in the document]
Comment (nikita1tup) [reply src/helpers/helpers.ts:250]: +

File: src/components/CopyText.vue [hunk 4]
```diff
@@ -65,10 +65,8 @@ export default defineComponent({
 <style scoped lang="scss">
 .text {
   @media (max-width: 480px) {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    width: 100px;
+    @include ellipsis();
```
Comment (ArtSuslov) [inline src/components/CopyText.vue:68]: use placeholders [when possible](https://medium.com/@linker21/scss-placeholders-vs-mixins-fe79b9dbb6d5)

**Review by ArtSuslov — CHANGES_REQUESTED** [review]

**Pull request merged**